Client/Clique/Common/Models/CreateEventEntries.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  CreateEventEntries.swift
+//  Clique
+//
+//  Created by Infinum on 30.11.2022..
+//
+
+import Foundation
+
+struct CreateEventEntries: Codable {
+    let name: String
+    let location: String
+    let timeStamp: String
+    let participantsNumber: String
```

Comment:
I think `participantsCount` would be a bit more better, no need to change it, but 😄